rs/replicated_state/src/replicated_state.rs

```diff
@@ -668,6 +668,18 @@ impl ReplicatedState {
         canisters_memory_usage + subnet_memory_usage
     }
 
+    /// Computes the memory taken by best-effort response messages.
+    pub fn best_effort_message_memory_taken(&self) -> NumBytes {
```

Comment:
It is somewhat confusing that we include the subnet queue usage here but then below don't shed from the queues? 

Should we maybe exclude the usage related to the subnet queues here and re-introduce it in the PR that will start shedding messages from the subnet queues?

Reply:
Done. Also extended the test to shed from the subnet queues.